js/widgets/behaviors/base.ts

```diff
@@ -243,7 +243,7 @@ export class NunjucksModel extends DynamicModel {
     }
 
     function handler(opts: any) {
-      const value = coercer(tmpl.render(opts));
+      const value = coercer(tmpl.render(opts).trim());
```

Comment:
@nrbgt : not sure if this is will impact performance, but if it does not, it helps read the rendered template when troubleshooting

Reply:
Resolved in [df42d1f](https://github.com/jupyrdf/ipyforcegraph/pull/124/commits/df42d1f49ef6a055a40aad25320f20fbe6a64e95)